src/app/containers/OnDemandHeading/index.jsx

```diff
@@ -82,7 +82,7 @@ const OnDemandHeadingContainer = ({
       id={idAttr}
       darkMode={darkMode}
       className={className}
-      {...(idAttr === 'content' && { tabIndex: '-1' })}
+      {...(idAttr === 'headline' && { tabIndex: '-1' })}
```

Comment:
I don't think we'd need the tabIndex anymore

Reply:
Nice catch! Thanks @jroebu14